test_cli_remapping/test/test_cli_remapping.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import functools
+import os
+import random
+import sys
+import time
+
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler
+from launch.launcher import DefaultLauncher
+import pytest
+import rclpy
+
+
+def get_environment_variable(name):
+    """Get environment variable or raise if it does not exist."""
+    path = os.getenv(name)
+    if not path:
```

Comment:
Nit: pedantically this should be `if path is None` as some environment can exist and be set but not trigger the if (like an empty string). In the context of this test, it may be more appropriate to expect valid paths and check for path existence rather than empty string or None

Reply:
I'll merge this PR as is since it adds test coverage, though in the future I hope to combine `test_cli_remapping` with `test_cli` which will be added in #274. To that end I fixed this in ec67875dd4c33264b8782fbb8abde6c0d4a552dc.

I'll create a ticket to merge the two after #274 is merged.